lib/src/main/cpp/headers/mkall.h

```diff
@@ -0,0 +1,463 @@
+/* DO NOT EDIT THIS FILE - it is machine generated */
```

Comment:
I am generally against committing generated files, but this is something we can probably fix later.